app/scripts/metamask-controller.js

```diff
@@ -1541,6 +1577,36 @@ module.exports = class MetamaskController extends EventEmitter {
     return '0x' + percentileNumBn.mul(GWEI_BN).toString(16)
   }
 
+  /**
+   * A method for retrieving of gas price from POA gas price oracles
+   *
+   * @returns {string} A hex representation of the suggested wei gas price.
+   */
+  getGasPriceFromOracles (networkId) {
+    return new Promise(async (resolve, reject) => {
+      const gasPriceOracleETC = 'https://gasprice-etc.poa.network'
+      const gasPriceOracleETH = 'https://gasprice.poa.network'
+      const gasPriceOralce = networkId === CLASSIC_CODE ?
```

Comment:
Here is a typo on the variable, it should be `gasPriceOracle`

Reply:
Thanks. Fixed.